src/eve/EveMeshOverlayEffect.js

```diff
@@ -1,4 +1,9 @@
 /**
+ * EveObjectChild
+ * @typedef {EveMeshOverlayEffect|EveChildExplosion|EveChildMesh|EveCurveLineSet|EveEffectRoot|EveTransform} EveObjectChild
```

Comment:
these are not exactly related objects:
EveMeshOverlayEffect goes to meshOverlays array,
EveTransform (being deprecated) lives in "children" array,
EveChild... are new things replacing EveTransform and they live in "effectChildren" array.
EveEffectRoot (also soon to be deprecated) is a top-level object